lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -371,7 +371,12 @@ void addFPBuiltinDecoration(SPIRVModule *BM, Instruction *Inst,
   }
 }
 
-SPIRVType *LLVMToSPIRVBase::transType(Type *T) {
+// IsBuiltin is set by caller to be true in two cases:
+// 1. Value associated with the type is a builtin variable.
+// 2. Value associated with the type is a GEPInst with address pointer as a
```

Comment:
Are both these cases hit by testing?

Reply:
After recent changes, we do not need to worry about the second case. The first one is covered.

Thanks